### ISSUE TITLE
Wrong show in search

### ISSUE THREAD

**Issue by john3voltas** [elgatito/plugin.video.elementum#848]
## Expected Behavior

Picking TV Show "House":
TV Shows >> Trakt > My Collection > House > Season 1 > Pilot
That should show a list of possible torrents to watch the pilot episode for House, M.D..
## Current Behavior

Picking TV Show "House":
TV Shows >> Trakt > My Collection > House > Season 1 > Pilot
That is showing a list of torrents to watch a lot of TV shows with the word House in their name but not the torrent for the pilot episode for House, M.D..
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used: 0.1.83

* Environment name and version: KODI 18.9

* Operating System and version: LibreELEC 9.2.8

* Link to Kodi debug log:

**Comment by dineshalphaque** [comment]
I had the exact problem with House, M.D. returning a bunch of shows which all had the word "House" in it. It's described in this thread :- https://github.com/elgatito/plugin.video.elementum/issues/841

TL;DR solution would be to search for the torrent either for the individual episode or the entire S1 and assign it to the TV show. i described it in the thread above. May need a restart of Elementum or Kodi to have the torrent assigned correctly if you're using Memory and not File as a storage type.

**Comment by john3voltas** [comment]
@dineshalphaque 
Thanks a bunch for your comment.
Could you please elaborate on how I can assign a torrent to a TV show? I can't find any reference on how to do so.
Many thanks.
Cheers

**Comment by antonsoroko** [comment]
it is in documentation https://elementum.surge.sh/context/

**Comment by elgatito** [comment]
@john3voltas That one is tricky. Try to search for movie "It" to see how disaster looks like.

**Comment by john3voltas** [comment]
Hi @elgatito 
I see your point.
But is there any way I can work around the issue.
Because it tried to associate a torrent with a given title (for House) and it says it can't find the torrent...

**Comment by antonsoroko** [comment]
@john3voltas 

>it says it can't find the torrent...

do you use memory or file storage?
assigning should work with both iirc, but right now i can vouch only for file storage. maybe with memory you need to restart kodi, like it was for **dineshalphaque**. might be some bug that i missed.

**Comment by antonsoroko** [comment]
@john3voltas i would appreciate a log file with failure of assigning torrent. 
http://elementum.surge.sh/contribute/log/

**Comment by elgatito** [comment]
This one is a special case.

Those are titles filled at themoviedb.org - https://www.themoviedb.org/tv/1408-house/titles
Original language for this show is "en" and original title is "House" (without M.D.), that is why those language names are send to Burst:
```
{'ar': 'هاوس', 'bg': 'д р хаус', 'br': 'dr. house', 'bs': 'house m.d.', 'ca': 'dr house', 'cn': '豪斯医生', 'cs': 'dr. house', 'cz': 'dr. house', 'de': 'dr. house', 'el': 'ιατρικές υποθέσεις', 'en': 'house', 'es': 'dr. house', 'fr': 'dr house', 'gb': 'house medical division', 'gr': 'ιατρικές υποθέσεις', 'he': 'האוס', 'hr': 'dr. house', 'hu': 'doktor house', 'il': 'האוס', 'it': 'dr. house medical division', 'ja': 'ドクター ハウス', 'jp': 'ドクター ハウス', 'ko': '하우스', 'kr': '하우스', 'lt': 'daktaras hausas', 'lv': 'doktors hauss', 'mx': 'dr. house', 'nl': 'house m.d.', 'original': 'house', 'pl': 'dr house', 'pt': 'dr. house', 'ro': 'dr. house', 'rs': 'др хаус', 'ru': 'доктор хаус', 'sa': 'هاوس', 'sk': 'dr. house', 'source': 'House', 'sr': 'др хаус', 'tw': '怪醫豪斯', 'ua': 'доктор хаус', 'uk': 'доктор хаус', 'us': 'house m.d', 'zh': '怪醫豪斯'}
```

'us' one looks more interesting for this one. but we also have 'gb' almost everywhere.

**Comment by antonsoroko** [comment]
just to clarify - i am asking for log file of failure of context menu "assign torrent" functionality.

**Comment by john3voltas** [comment]
@antonsoroko 
Hi. Thanks for trying to help.
Hadn't noticed you were offering help on the "assign torrent" issue.
I'll get home late today, but tomorrow I'll post a log.
Thanks in advance.
Cheers

**Comment by john3voltas** [comment]
Hi @antonsoroko 
Sorry for the delay.
As promised, here is the debug log. 

https://paste.kodi.tv/ubunawolus  

Cheers

**Comment by antonsoroko** [comment]
@john3voltas 
thank you for log.

just curious:
if after failed attempt to assign torrent you start video (to which you tried to assign torrent):
1. immediately 
2. after restart kodi
will your elementum say that there is a torrent assigned to your video?

**Comment by john3voltas** [comment]
@antonsoroko 
No, unfortunately Elementum always tells me 'torrent not found'. And whether I try to immediately start the video or if I restart Kodi and try to start the video, Elementum will always try to locate a torrent file using Elementum burst's trackers...

**Comment by JordanPlayz158** [comment]
This is admittedly a bad solution but... it was also called "House, M.D." which (at least on 1337x) seems to clean up the results and the top 10 results were actually about "House" so maybe for certain shows (I hate hardcoded but don't know of any way to automate it) you could have alternative names to look for.

**Comment by john3voltas** [comment]
This is so frustrating...
I know this is the only TV show I can't watch and that I should be awfully glad that Elementum works on all the other shows. But still, very very frustrating...

**Comment by elgatito** [comment]
@rdscorreia74 Did you try the latest version of Elementum? I recall I was doing changes to properly find "House M.D.".
